Formula/chartmuseum.rb

```diff
@@ -0,0 +1,43 @@
+class Chartmuseum < Formula
+  desc "Host your own Helm Chart Repository"
+  homepage "https://chartmuseum.com"
+  url "https://github.com/helm/chartmuseum/archive/v0.9.0.tar.gz"
+  sha256 "e5fed41b098141a56037340ad3b0857c7defae6d143af56b6003a270cc75631f"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GO111MODULE"] = "on"
+    ENV["GOPROXY"] = "https://gocenter.io"
+
+    system "go",
+        "build",
+        "-v",
+        "-ldflags",
+        "-X main.Version=#{version} -X main.Revision=9382222",
```

Comment:
Is that revision always correct?

Reply:
No, it will change for different versions. I will fix it by using git url to download